arches_for_science/templates/views/components/resource-instance-creator.htm

```diff
@@ -0,0 +1,227 @@
+                                                        $data.inverseOntologyProperty,
+                                                        $data.ontologyClass(),
+                                                        self.rootOntologyClass,
+                                                        $root.translations.selectAnOntologyProperty
+                                                    )
+                                                },
+                                                attr: {'aria-labeledby': rrToResourceLabel},
+                                            "
+                                        ></select>
+                                    </span>
+
+                                    <!-- Instance name -->
+                                    <span class="col-xs-12" data-bind="text: self.resourceInstanceDisplayName + ' (' + self.makeFriendly(self.rootOntologyClass) + ')'">
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- /ko -->
+    </div>
+
+    <div data-bind="if: !!value() && value().hasOwnProperty('length') && value().length > 1">
+        <div class='file-workbench-filecount' data-bind='text: value().length + $root.translations.relationships'></div>
+    </div>
+
+</div>
+<!-- /ko -->
+<!-- /ko -->
+
+<div class="create-resource-instance-card-component rr-table-pop" id="resource-creator-panel"
+    data-bind="with: newResourceInstance, style: {transform: !!newResourceInstance() ? 'translate(0,0)' : 'translate(100%,0)'}">
+    <!-- ko component: {
+        name: 'related-instance-creator',
+        params: {...$data, graphName: $parent.graphLookup[graphid].name}
+    } --><!--/ko -->
+</div>
+
+<div class="create-resource-instance-card-component rr-table-pop" id="resource-report-panel"
+    data-bind="style: {transform: !!reportResourceId() ? 'translate(0,0)' : 'translate(100%,0)'}">
+
+    <!-- ko if: reportResourceId -->
+    <div class="create-instance-header" style="display: flex; justify-content: space-between;">
+        <span data-bind="text: $root.translations.relatedResourceSummary"></span>
+
+        <div data-bind="click: function(){reportResourceId(null)}, clickBubble: false" class="close-new-step">
+            <button class="btn btn-labeled btn-lg fa fa-arrow-right btn-primary">
+                <span data-bind="text: $root.translations.return"></span>
+            </button>
+        </div>
+    </div>
+    <div class="create-instance-panel" data-bind="component: {
+        name: 'resource-report-abstract',
+        params: {
+            resourceid: reportResourceId(),
+            summary: true,
+        }
+    }"></div>
+    <!-- /ko -->
+</div>
+<button class="btn btn-primary" data-bind="click: function(){newResource();}">{% trans "Add Resource" %}</button>
```

Comment:
I can imagine we might want to use this elsewhere in the app, in which case 'Add Resource' is a good button label. But for this round of work, maybe 'New Physical Thing' would be clearer?